libs/sumologic/sumotf/sumotf.go

```diff
@@ -23,6 +23,7 @@ type SLOLibrarySLO struct {
 	ID              string `json:"id,omitempty"`
 	Name            string `json:"name"`
 	Description     string `json:"description"`
+	Tags            string `json:"tags"`
```

Comment:
why is this a `string` and not a `map[string]string` ?

Reply:
serializing `map[string]string` to string would produce something of this form: `key: value`, while the terraform spec expects `=` instead of `:`

Reply:
If the output is a file in hcl (terraform) syntax, using json marshal to do that in the first place is a bit wierd. Either we could write a serializer on our own, or we could use something like https://pkg.go.dev/github.com/hashicorp/hcl/v2/hclwrite